playground/src/components/echo-server/echo-server.js

```diff
@@ -8,8 +8,23 @@ import Text from '@commercetools-uikit/text';
 import PrimaryButton from '@commercetools-uikit/primary-button';
 import { CodeBlock } from '@commercetools-docs/ui-kit';
 
+const initialState = {
+  isLoading: false,
+  result: null,
+};
+const reducer = (state, action) => {
+  switch (action.type) {
+    case 'loading':
+      return { isLoading: true, result: null };
+    case 'complete':
+      return { isLoading: false, result: action.payload };
+    default:
+      return state;
```

Comment:
...or `throw new Error(Action of type `${action.type}` not implemented.)`?

Reply:
Hmm why? The `default` case returns the inital state.